src/api_or_sql/funcs.php

```
@@ -57,6 +57,38 @@ function exists_by_qids_query($lang, $category)
+        SELECT a.qid, a.title, a.category, t.code, t.target
+            FROM all_qids_titles a
+            JOIN all_qids_exists t
+            ON t.qid = a.qid
+        WHERE t.code = ?
+
+        AND (t.target != '' AND t.target IS NOT NULL)
+    SQL;
+    // ---
+    $params = [$lang];
+    // ---
+    $u_data = super_function($api_params, $params, $query, $no_refind = false, $table_name = "all_qids_titles");
```

PHPMD (2.15.0)

Avoid unused local variables such as '$no_refind'. (undefined)

(UnusedLocalVariable)